pontoon/actionlog/models.py

```diff
@@ -25,7 +25,7 @@ class ActionLog(models.Model):
     action_type = models.CharField(max_length=50, choices=ACTIONS_TYPES)
     created_at = models.DateTimeField(auto_now_add=True)
     performed_by = models.ForeignKey(
-        "auth.User", models.CASCADE, related_name="actions"
+        "auth.User", models.SET_NULL, related_name="actions", null=True
```

Comment:
That shouldn't be necessary. If Django still tries to delete related objects (same for Comments and Terms), the custom delete code probably doesn't execute. Maybe you need to add an [action](https://docs.djangoproject.com/en/3.0/ref/contrib/admin/actions/#adding-actions-to-the-modeladmin)?